Mathlib/Algebra/Tropical/Basic.lean

```diff
@@ -349,11 +344,14 @@ theorem add_eq_right_iff {x y : Tropical R} : x + y = y ↔ y ≤ x := by
   rw [trop_add_def, trop_eq_iff_eq_untrop, ← untrop_le_iff, min_eq_right_iff]
 #align tropical.add_eq_right_iff Tropical.add_eq_right_iff
 
-@[simp]
+/- Porting note: `simp` can prove it, but only by rewriting entire equality, and not by just
+simplify the LHS -/
+@[simp, nolint simpNF]
```

Comment:
I think Gabriel explained that the linter was actually right, didn't he?
```suggestion
```
https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/simp.20can.20prove.20it/near/319474938